src/java.base/unix/native/libjli/java_md.c

```diff
@@ -376,12 +376,35 @@ CreateExecutionEnvironment(int *pargc, char ***pargv,
          *     o          $JVMPATH (directory portion only)
          *     o          $JRE/lib
          *     o          $JRE/../lib
-         *     o          ZLIBNX_PATH (for AIX P9 or newer systems with NX)
+         *     o          ZLIBNX_PATH (for AIX P9 or newer systems with NX, unless -XX:-UseZlibNX is set)
          *
          * followed by the user's previous effective LD_LIBRARY_PATH, if
          * any.
          */
 
+#ifdef AIX
+        int argc = *pargc - 1; // skip the launcher name
+        char **argv = *pargv + 1;
```

Comment:
Sorry, I should have noticed in https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/664, but `argc` and `argv` are already declared on lines 307 and 308. Can you choose new variable names (here and in the related PRs and create an update for jdknext)?

Reply:
Created https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/668. Once approved I will backport it.